File: src/test/java/ru/spb/se/contexthelper/ui/StackExchangeThreadsTreeCellRenderTest.java
package ru.spb.se.contexthelper.ui;

import org.junit.Test;

import static com.google.common.truth.Truth.assertThat;

import static ru.spb.se.contexthelper.ui.StackExchangeThreadsTreeCellRender.convertScoreToString;

public class StackExchangeThreadsTreeCellRenderTest {
  @Test
  public void convertPositiveScoreToString() {
    String formatted = convertScoreToString(19);
    assertThat(formatted).isEqualTo("+19");
  }

  @Test
  public void convertZeroScoreToString() {
    String formatted = convertScoreToString(0);
    assertThat(formatted).isEqualTo("+0");
  }

  @Test
  public void convertNegativeScoreToString() {
    String formatted = convertScoreToString(-736);
    assertThat(formatted).isEqualTo("-736");
  }
}

File: src/test/java/ru/spb/se/contexthelper/ui/StackExchangeThreadsTreeCellRenderTest.kt
package ru.spb.se.contexthelper.ui

import org.junit.Test

import com.google.common.truth.Truth.assertThat
import ru.spb.se.contexthelper.ui.StackExchangeThreadsTreeCellRender.Companion.convertScoreToString

class StackExchangeThreadsTreeCellRenderTest {
    @Test
    fun convertPositiveScoreToString() {
        val formatted = convertScoreToString(19)
        assertThat(formatted).isEqualTo("+19")
    }

    @Test
    fun convertZeroScoreToString() {
        val formatted = convertScoreToString(0)
        assertThat(formatted).isEqualTo("+0")
    }

    @Test
    fun convertNegativeScoreToString() {
        val formatted = convertScoreToString(-736)
        assertThat(formatted).isEqualTo("-736")
    }
}
